src/shared-components/tooltip/__snapshots__/test.tsx.snap

```diff
@@ -59,9 +59,20 @@ exports[`<Tooltip /> UI snapshot renders content and children 1`] = `
   margin-left: -10px;
 }
 
+.emotion-4 {
```

Comment:
The application of `emotion-4` on the `svg` in the snapshot indicates there's no regression with the behavior we were achieving with the custom `@svgr` template, that put the `css` prop directly on the `<svg>` element.